Dockerfile

```diff
@@ -1,155 +1,143 @@
-FROM node:18-slim as base
+# base
+# ------------------------------------------------
+FROM node:18-bookworm-slim as base
```

Comment:
We're using bookworm, a "codename" for a version of a Debian LTS release, because we think it's a best practice to pin the LTS of the OS we're using like we do for Node.js